From 959359c76b9e64b26cbc6ae3e21f57e5953d3822 Mon Sep 17 00:00:00 2001
From: Saleem Haveri <shaveri@accuratebackground.com>
Date: Tue, 21 Jun 2016 02:43:01 -0700
Subject: [PATCH] core 1.0

---
 HelloWorld.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/HelloWorld.java b/HelloWorld.java
index cc08bdc..e13740a 100644
--- a/HelloWorld.java
+++ b/HelloWorld.java
@@ -3,5 +3,7 @@ public class HelloWorld {
     public static void main(String[] args) {
         // Prints "Hello, World" to the terminal window.
         System.out.println("Hello, World");
+		//core changes
+		StringBuilder b = new StringBuilder();
     }
 }
-- 
2.9.0.windows.1

